### PR TITLE
Adding Team Page

### PR DESCRIPTION
**Pull request by mienaikoe**
Adding a Team Page that includes upcoming events, past events for coach
and the roster of the team.

### CODE REVIEW

File: client/pages/Team/EventCard.css [hunk 1]
```diff
@@ -17,14 +19,20 @@
   padding: 0 2rem;
 }
 
+.cardTitle{
+  font-family: 'Oswald', sans-serif;
+  font-size: 28px !important;
```
[the remark this answers is not in the document]
Comment (mienaikoe) [reply client/pages/Team/EventCard.css]: No... react-toolbox is really sloppy with their themes, and they would often set things that are a nuisance to override without `!important`.

[the remark this answers is not in the document]
Comment (k3ntako) [reply client/pages/Team/EventCard.css]: I actually ran into that last night too 😕

[the remark this answers is not in the document]
Comment (mienaikoe) [reply client/pages/Team/EventCard.css]: Yea sometimes it's necessary. Also, you were working on this last night??

[the remark this answers is not in the document]
Comment (k3ntako) [reply client/pages/Team/EventCard.css]: Just a tiny bit!

**Pull request merged**